tensorflow/python/ops/math_ops.py

```diff
@@ -3204,6 +3204,16 @@ def sigmoid(x, name=None):
   """Computes sigmoid of `x` element-wise.
 
   Specifically, `y = 1 / (1 + exp(-x))`.
```

Comment:
I'd suggest changing this line to

Specifically, computes `1 / (1 + exp(-x))`.

or something of that nature as a result of @mihaimaruseac's new review.